src/components/CommitChartLabel/index.tsx

```diff
@@ -58,7 +56,7 @@ const CommitChartLabel: React.VFC<Props> = ({
         >
           {shortenGithash(githash)}
         </StyledRouterLink>{" "}
-        <b>{shortDate(createDate)}</b>
+        <b>{getDateCopy(createDate)}</b>
```

Comment:
why do we format with the user timezone below but not here?

Reply:
i'm inclined to advocate for the opposite since it makes it more confusing if dates are in different timezones through out the application.

Reply:
Let's discuss in the design sync before moving forward